.github/workflows/codeql.yml

```diff
@@ -0,0 +1,39 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "develop", "master", "hotfix/*", "release/*", "feature/**" ]
```

Comment:
Any reason not to to make it `*`, in case somebody doesn't use Hubflow/follow the convention of creating a `feature` branch?

Reply:
@kathy-t found https://github.com/dockstore/dockstore-ui2/pull/1649#discussion_r1028137161
It does seem to be more applicable.